snuba/datasets/cdc/message_filters.py

```diff
@@ -31,9 +31,7 @@ def should_drop(self, message: Message[KafkaPayload]) -> bool:
         )
 
         if table_name:
-            table_name = table_name.decode("utf-8")
-            if table_name != self.__postgres_table:
-                metrics.increment("cdc_message_dropped", tags={"table": table_name})
+            if table_name.decode("utf-8") != self.__postgres_table:
```

Comment:
Why not
`return table_name and table_name.decode("utf-8") != self.__postgres_table`

now that there is no other statement in this method?

Reply:
Good point, will tidy this up.